test_api_crud.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+
+def test_get_user():
+    p = {"page": 2}
+    resp = requests.get("https://reqres.in/api/users", params=p)
+    json_response = resp.json()
+
+    print(json_response["total_pages"])
```

Comment:
it is better to use logger for info messages ex. logging.info(json_response["total_pages"]), print is not a good practice

Reply:
done